### ISSUE TITLE
[TW-41] Tasks in subprojects are not counted in project completion

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#103]
_Renato Alves on 2014-01-22T20:04:10Z says:_ 

 As a follow up of the discussion in IRC:

        jck | Tasks in subprojects are not counted in project completion, is that a bug?
pbeckingham | jck: Arguable.  Maybe.
pbeckingham | Should probably be written up, and discussed.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-01-22T20:04:10Z
Modified: 2015-02-28T14:34:30Z
```

**Comment by taskwarrior** [comment]
_Tomas Babej on 2015-02-28T14:34:30Z says:_ 

 Fixed with: 48be6986c27ee5a6734a544be01e2cb5c66463b3
